core/src/test/java/io/undertow/server/security/ClientCertRenegotiationTestCase.java

```diff
@@ -67,7 +67,7 @@ protected List<AuthenticationMechanism> getTestMechanisms() {
 
     @BeforeClass
     public static void startSSL() throws Exception {
-        DefaultServer.startSSLServer(OptionMap.create(SSL_CLIENT_AUTH_MODE, NOT_REQUESTED));
+        DefaultServer.startSSLServer(OptionMap.create(SSL_CLIENT_AUTH_MODE, REQUESTED));
```

Comment:
Can be changed SSL_CLIENT_AUTH_MODE to REQUESTED?

Reply:
@kstekovi do you know the reason why this started to be problem with JDK17 and why it works as expected with older JDKs?

Reply:
TBH - I'm not sure whether this change is actually correct. We need more investigation why this behavior changed in JDK17. I've put some comment to the original Jira.